tests/orion/database/test_alembic_commands.py

```diff
@@ -69,3 +71,15 @@ def test_alembic_stamp(self, mocked):
         _, kwargs = mocked.call_args
         assert mocked.call_count == 1
         assert kwargs["revision"] == "abcdef"
+
+    async def test_concurrent_upgrade(self):
+        jobs = [run_sync_in_worker_thread(alembic_upgrade) for _ in range(0, 10)]
+        await asyncio.gather(*jobs)
+
+    @pytest.mark.flaky(max_runs=3)
```

Comment:
Does this need to be flaky?

Reply:
I had it `range(0, 3)` and it timed out; let me see if 2 times out.